rnote-engine/src/pens/shaper.rs

```diff
@@ -55,6 +56,8 @@ pub struct Shaper {
     #[serde(rename = "rough_options")]
     pub rough_options: RoughOptions,
 
+    #[serde(skip)]
+    pub constraint: Constraint,
```

Comment:
when Serialize and Deserialize is derived, this field does not need to be skipped. Also singular -> plural